### PR TITLE
Added get playlist handling to spotifywrapper

### PR DESCRIPTION
**Pull request by FranekDev**
Added get playlist handling to spotifywrapper as well as test method for getPlaylist().

### CODE REVIEW

**Comment by mati2251** [comment]
#54

File: backend/src/test/kotlin/pl/akai/fillist/web/spotifywrapper/SpotifyPlaylistsTests.kt [hunk 1]
```diff
@@ -52,4 +52,18 @@ class SpotifyPlaylistsTests {
         // assertEquals(playlist.description, createPlaylistRequestBody.description)
         assertEquals(playlist.owner.id, userId)
     }
+
+    @Test
+    fun getPlaylist() {
+        // Sample playlist ID
```
Comment (mati2251) [inline backend/src/test/kotlin/pl/akai/fillist/web/spotifywrapper/SpotifyPlaylistsTests.kt:58]: Comments quickly become outdated, making them less helpful for understanding code. They can also make code more difficult to understand, especially if they are not updated as the code changes. If you look at the code below, you know what that stuff makes. In my opinion these comment is useless.

File: backend/src/main/kotlin/pl/akai/fillist/web/spotifywrapper/playlists/SpotifyPlaylistsService.kt [hunk 2]
```diff
@@ -2,6 +2,7 @@ package pl.akai.fillist.web.spotifywrapper.playlists
 
 import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.stereotype.Service
+import org.springframework.web.bind.annotation.PathVariable
```
Comment (mati2251) [inline backend/src/main/kotlin/pl/akai/fillist/web/spotifywrapper/playlists/SpotifyPlaylistsService.kt:5]: https://github.com/akai-org/fillist/security/code-scanning/5146
Fix alert

**Review by mati2251 — CHANGES_REQUESTED** [review]

File: backend/src/test/kotlin/pl/akai/fillist/web/spotifywrapper/SpotifyPlaylistsTests.kt (hunk 1 above)
Comment (FranekDev) [reply backend/src/test/kotlin/pl/akai/fillist/web/spotifywrapper/SpotifyPlaylistsTests.kt]: Agree, I removed redundant comment.

File: backend/src/main/kotlin/pl/akai/fillist/web/spotifywrapper/playlists/SpotifyPlaylistsService.kt (hunk 2 above)
Comment (FranekDev) [reply backend/src/main/kotlin/pl/akai/fillist/web/spotifywrapper/playlists/SpotifyPlaylistsService.kt]: Removed unused import

**Review by mati2251 — APPROVED** [review]

**Pull request merged**